tests/orquestra/integrations/qiskit/conversions/circuit_conversions_test.py

```diff
@@ -532,14 +532,6 @@ def test_binding_and_exporting_parametrized_circuit_results_in_equivalent_circui
             f"equal to\n{_draw_qiskit_circuit(ref_bound)}"
         )
 
-    def test_converting_circuit_with_daggers_fails_explicitly(self):
```

Comment:
[issue] You added some new logic which fixes some failure modes we observed. Those failures were not covered by tests, so we learned about them "in the wild". Why you have not added those cases as test cases here?

Reply:
I'm not sure what made the circuit conversion that was found "in the wild" fail. So I couldn't add a test which checked it. I just know that it worked now. I can use previous versions to find out what the error was. So I suppose I'll do that now and add a test.